workitem/workitem_revision_repository.go

```diff
@@ -35,8 +35,9 @@ type GormRevisionRepository struct {
 // Create stores a new revision for the given work item.
 func (r *GormRevisionRepository) Create(ctx context.Context, modifierID uuid.UUID, revisionType RevisionType, workitem WorkItem) error {
 	log.Info(nil, map[string]interface{}{
-		"pkg":              "workitem",
-		"ModifierIdentity": modifierID,
+		"pkg":               "workitem",
+		"modifier_identity": modifierID,
+		"revision_type":     revisionType,
```

Comment:
Buff ! It isn't you but we have here a mix of camel case and snake case fields that we pass to the logs. I think we might agree to one notation. I think that would be snake case.
@aslakknutsen @kwk  thoughts ?